docs/source/configuration/environmentvariables.md

```diff
@@ -92,10 +92,10 @@ It will enable the log of missing i18n messages (in console).
 
 ### Add add-ons via environment variable
 
-Quite often, you need different configurations and enabling components (for example, testing purposes).
-You can use the `ADDONS` environment variable to define them.
-They sum up to the ones set in `package.json` or programatically in `volto.config.js`.
+You can use the `ADDONS` environment variable to enable and configure add-ons in your project.
+They sum up to the ones set fixed in `package.json` or programatically in `volto.config.js`.
```

Comment:
💡  Now I understand!

```suggestion
They are equivalent to those defined in `package.json`, or set programatically in `volto.config.js`.
```

Reply:
Nope... not the intention! So the resultant ADDONS are package.json addons key + volto.config.js ones + the ones in the env. variable. How do we state it?